script.js

```diff
@@ -6,4 +9,317 @@ function openNav(){
+                    //doc.cookie --> /season_config/season/elementName=elementValue;...
+
+                    var newTRHtml = "<tr><td>" + newElementName + 
+                    "</td><td>" + newElementPoints + "</td></tr>";
+                    
+                    $("#scoringTable").append(newTRHtml);
+                    $("#elementConfirmationBox").html('<i>Element "' + newElementName + '" added.<\i>');
+
+                    $("#elementKey").val("");
+                    $("#elementPoints").val("");
+                } else
+                {
+                    $("#elementConfirmationBox").html('<i>Element "' + newElementName + '" already exists.<\i>');
+                }
+            } else
+            {
+                $("#elementConfirmationBox").html("<i>Please check a season.<\i>");
+            }
+        } else
+        {
+            $('#elementConfirmationBox').html("<i>Please fill in both fields.<\i>");
+        }      
+    });
+
+
+    //start of questionnaire js code
+    $("#pitScoutingSubmit").click(function(){
+        var allQuestions = $("#pitScoutingTextBox").val().split('\n');
+
+        var currSeason = getCookie("currCheckedSeason");
+        console.log(allQuestions);
+        //console.log(currSeason);
+        //console.log(currSeason);
+        //console.log(questionsArr);
+      
+        if (currSeason === "null" || currSeason ===" ")
+        {
+            $("#pitScoutConfirmationBox").html("<i>Please check a season.<\i>");
+        } else
+        {
+            for (var i = 0; i < allQuestions.length;i++)
+            {
+                // pit scouting cookies should be in the format:
+                // (currCheckedSeason) pitScout Q(i): question; expiry date...
+                var question = allQuestions[i].toString();
+                var cookieName = currSeason + " pitQuestions";
+                if (question != "")
+                {
+                    if (!getCookie(cookieName).includes(question))
+                    {
+                        //cookie --> {season} questions = {q1},{q2},{q3}
+                        document.cookie = cookieName + '=' + getCookie(cookieName) + question + ',; expires=Wed, 1 Jan 2038 12:00:00 UTC; path=/';
+                        var confirmationText = '<i>Question "' + question + '" saved.<\i><br>';
+                        if (!$("#pitScoutConfirmationBox").html().includes(confirmationText))
+                        {
+                            $("#pitScoutConfirmationBox").append(confirmationText);
+                        }
+                    } else
+                    {
+                        var confirmationText = '<i>Question "' + question + '" already exists.<\i><br>';
+                        if (!$("#pitScoutConfirmationBox").html().includes(confirmationText))
+                        {
+                            $("#pitScoutConfirmationBox").append(confirmationText);
+                        }
```

Comment:
And, what if I want to clear one out?

For that matter, with reloading the text box to show already stored ones - this could get really annoying if I have 5 already and want to add one.

Reply:
I see that if I clear the text box and add the one I want to add that I get back all 6, but what if I want to edit a line? current implementation is going to store both versions and reload them in order.

Should probably just make this store whatever is in the text box right now - that allows for updating, deleting, and adding without any major work (think of it like a text document - when the user hits save -they expect exactly that to be saved - not appended to the last version).